lib/src/components/buttons/details_button.dart

```diff
@@ -34,16 +39,27 @@ class DetailsButton extends StatelessWidget {
           children: [
             Text(
               title,
-              style: const TextStyle(
-                letterSpacing: -0.9,
-                fontSize: 19.2,
-                fontWeight: FontWeight.w500,
-              ).copyWith(
+              style: TextStyle(
+                fontFamily: 'Roboto',
+                fontSize: fontSize,
+                fontWeight: FontWeight.w600,
+                letterSpacing: letterSpacing,
                 color: color,
               ),
             ),
             const SizedBox(width: 2),
-            Icon(SFSymbols.chevron_down, color: color, size: iconSize),
+            Container(
+              padding: const EdgeInsets.only(top: 4.0),
```

Comment:
It only sets a padding -- let's use the appropriate Widget for that
```suggestion
            Padding(
              padding: const EdgeInsets.only(top: 4.0),
```